tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -213,13 +213,14 @@ When('the user uploads a created file {string} using the webUI', function (eleme
   return client.page.personalPage().uploadFile(uploadPath)
 })
 
-When('the user uploads a created file {string} with overwrite using the webUI', function (element) {
-  const uploadPath = path.join(client.globals.mountedUploadDir, element)
-  client.page
-    .personalPage()
-    .selectFileForUpload(uploadPath)
-    .then(() => client.page.personalPage().confirmFileOverwrite())
-})
+When(
+  'the user uploads a created file {string} with overwrite using the webUI',
+  async function (element) {
+    const uploadPath = path.join(client.globals.mountedUploadDir, element)
+    await client.page.personalPage().selectFileForUpload(uploadPath)
+    await client.page.personalPage().confirmFileOverwrite()
+  }
+)
```

Comment:
I just want to know do these changes make any difference.

Reply:
Yes, with my findings from debugging I can say that even if you just use the `async and await`, it works, but I added a wait for `Upload complete` dialog box too to be more sure that the file uploaded to the server successfullly.

Reply:
The code is now modified though.